src/ResourceManager/ApiManagement/Commands.ApiManagement.Test/packages.config

```diff
@@ -14,6 +14,8 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
Remove Microsoft.Rest.ClientRuntime and Microsoft.Rest.ClientRuntime.Azure, as these are imported by common dependencies.